.bumpversion.cfg

```diff
@@ -1,21 +1,13 @@
 [bumpversion]
-parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)-?(?P<release_type>[dev|rc|stable]*)(?P<iteration>\d*)
-serialize = {major}.{minor}.{patch}-{release_type}{iteration}
-	{major}.{minor}.{patch}-{release_type}
-	{major}.{minor}.{patch}
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)
+serialize = {major}.{minor}.{patch}
+	{major}.{minor}
 tag = True
 commit = True
 message = Cut New Release: {current_version} → {new_version}
-current_version = 0.3.11
+current_version = 0.3.11-dev1
```

Comment:
wouldn't you want to change current_version to a `major.minor.patch` ?

Reply:
Circle should handle that when bumping